releasetasks/templates/desktop/publish_balrog.yml.tmpl

```diff
@@ -1,5 +1,5 @@
+{% if not release_eta %}
```

Comment:
There may be something ( I don't know TBH). I thought about using the same routes for both tasks, but it won't probably help when we have `release_eta` set - it's hard to predict the actual sign off times, esp if they happen after `release_eta`.